objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloQueriesLocationMapper.java

```diff
@@ -45,7 +45,7 @@
     static final String PB_FILE = "coco-" + COCO_MODEL_VERSION + ".pb";
     static final String META_FILE = "coco-" + COCO_MODEL_VERSION + ".meta";
     static final String OUTPUT_DIR = System.getProperty("buildDir") + "/resources/out";
-    static final String IP_CAMERA_ADDRESS = "http://207.192.232.2:8000/mjpg/video.mjpg";
+    static final String IP_CAMERA_ADDRESS = "http://61.208.190.221:8001/cgi-bin/camera?resolution=640&quality=1&Language=0&1639432689";
```

Comment:
Same as above. Could you also have a central file that holds this value, so it only needs to be changed once when a camera is fixed?

Reply:
I'll add an issue and add a properties file or something with the camera to use.  There are some places where we either don't want or don't care about detecting things but others where we do.  So there are some reasons to have a couple of them.

Reply:
This is now issue #298